gradio/components.py

```diff
@@ -3235,6 +3235,22 @@ def postprocess(
         """
         if y is None:
             return None
+        if isinstance(y, dict):
```

Comment:
Implementation looks good to me but let's add some unit tests for this please. Some helpful cases would be no entities, all entities, example in the demo.

Reply:
Done